src/main.cpp

```diff
@@ -355,6 +357,7 @@ int main(int argc, char *argv[]) {
 		exit(1);
 	}
 
+    packer.pack(output_format_version);
```

Comment:
Indentation looks incorrect. I would also put the definition next to the use, and add a comment:

```
// This version number should be incremented whenever the output format
// is changed in a way which will break existing parsers.
const int output_format_version = 0;
packer.pack(output_format_version);
```